### PR TITLE
Included 'alt' attribute to the images

### LINKED ISSUE #19334: Ensure no missing `alt` attributes
### MDN URL

https://developer.mozilla.org/en-us/learn/css/css_layout/flexbox/

### What specific section or headline is this issue about?

All of MDN content

### What information was incorrect, unhelpful, or incomplete?

333 `alt` attributes are missing. 29 have alt attributes that are the image file name. Untold others have unhelpful alt attributes.

### What did you expect to see?

Every image in MDNs content should have a useful alt attribute (other than tutorials explaining bad practices, maybe)>

### Do you have any supporting links, references, or citations?

List  of missing alt attributes: https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0

### Do you have anything more you want to share?

When writing an alt attribute, think about how you would describe the image to someone who could read the page's content but couldn't load any images. What information does the image convey that is important and relevant to the user?

For example, if there are green boxes with a check mark for good and a red box with an X for bad, the user doesn't need to know the color of the box or the check or X. They do need to know what it is and if it's good or bad.

If there is a screenshot, explain the reason that image was used, not the contents of the screenshot. Don't provide information they don't need or already know. For example, if you're on a page about changing settings on Duck Duck go, if you have a screenshot of duck-duck-go search result, you don't need to explain what the search term is, how many results there are, how long the search took, etc. as that is not the point of the image. Since you're showing how to change settings in duck-duck-go, the alt might be "The settings icon is in the navigation bar below the search field." We didn't include "screenshot" or "duck duck go" as the user doesn't need to know it's a screenshot and already knows it's duck duck go as they are on a page explaining changing duck-duck-go settings.

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Summary

Added 'alt' attributes to various images present in the page.

#### Related issues

Part of #19334 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Abhiram-Gullapalli** [comment]
@estelle Could you review this

File: files/en-us/learn/server-side/django/forms/index.md [hunk 1]
```diff
@@ -462,15 +462,15 @@ You can alternatively manually construct a test URL like this — `http://127.0.
 
 If you are successful, the default form will look like this:
 
-![](forms_example_renew_default.png)
+![Default form which displays the book details, due date, renewal date and a submit button appears in case the link works successfully](forms_example_renew_default.png)
 
 The form with an invalid value entered will look like this:
 
-![](forms_example_renew_invalid.png)
+![Image displaying output in case an invalid date is entered in the form](forms_example_renew_invalid.png)
```
Comment (estelle) [inline files/en-us/learn/server-side/django/forms/index.md:469]:
```suggestion
![Same form as above with an error message: invalid date - renewal in the past](forms_example_renew_invalid.png)
```

File: files/en-us/learn/server-side/django/forms/index.md [hunk 2]
```diff
@@ -462,15 +462,15 @@ You can alternatively manually construct a test URL like this — `http://127.0.
...
 
 The list of all books with renew links will look like this:
 
-![](forms_example_renew_allbooks.png)
+![Displays list of all renewed books along with their details](forms_example_renew_allbooks.png)
```
Comment (estelle) [inline files/en-us/learn/server-side/django/forms/index.md:473]:
```suggestion
![Displays list of all renewed books along with their details. Past due is in red.](forms_example_renew_allbooks.png)
```

File: files/en-us/learn/server-side/django/forms/index.md [hunk 3]
```diff
@@ -652,13 +652,13 @@ You can test editing records by appending _/update/_ to the end of the detail vi
 
 Finally, we can delete the page by appending delete to the end of the author detail-view URL (e.g. `http://127.0.0.1:8000/catalog/author/10/delete/`). Django should display the delete page shown below. Press "**Yes, delete.**" to remove the record and be taken to the list of all authors.
 
-![](forms_example_delete_author.png)
+![Screenshot displaying an option to delete author](forms_example_delete_author.png)
```
Comment (estelle) [inline files/en-us/learn/server-side/django/forms/index.md:655]:
```suggestion
![Form with option to delete author](forms_example_delete_author.png)
```

**Review by estelle — COMMENTED** [review]
Thanks. I made a few edits to include additional information the images display

**Review by estelle — APPROVED** [review]
Thank you!

**Pull request merged**